src/content/docs/ecosystem/validators/overview.mdx

```diff
@@ -3,20 +3,120 @@ title: SKALE Validators
 are run on all or a subset [multi-tenancy] of each node’s computation and storage resources.
 
-As each node in the network continues to participate in their assigned SKALE Chains, they are each awarded
-bounties based upon their performance at the end of each network epoch. Each node is be monitored by their peer
+As each node in the network continues to participate in their assigned SKALE Chains, they are awarded
+bounties based on their performance at the end of each network epoch. Each node is monitored by their peer
 nodes. When a SKALE Chain has reached the end of its lifetime, the resources (computation, storage) of its
 virtualized sub-nodes will be freed so that validator nodes may participate in newly created SKALE Chains.
 
-Validator nodes also receive direct payment from SKALE Chain pricing which is "rent" paid by the chain owners/operators
-for this compute.
\ No newline at end of file
+Validator nodes also receive direct payment from SKALE Chain pricing, which is "rent" paid by the chain owners/operators
+for this compute.
+
+## SKALE Economics
+
+Please explore the following links to learn more about the SKALE Token Economics:
+
+- [SKALE Validator & Delegator Token Economics](https://skale.network/blog/validator-economics)
+- [Economics FAQ](https://skale.network/blog/skale-update-economics-faq)
+- [Token supply chart](https://supply.skale.network/supply/index.html)
+- [Metrics that matter blog](https://skale.network/blog/metrics-that-matter-for-token-launches)
+- [Tokenomics one-pager](https://skale.network/tokenomics)
+- [Token page](https://skale.network/token)
+
+## Expectations
+
+- Reliable connection to SKALE Network (low latency, high uptime)
+- Robust security practices and network architecture
+- Running the latest SKALE Node software releases
+- Participation in governance (when applicable)
+
+To encourage proper behavior amongst network participants, SKALE follows a Proof of Stake (PoS) system whereby each node must
+stake a predetermined amount of SKALE tokens to be slashed at the citation of any activity not condoned by the network.
+
+Activities not approved by the network include those which denote a failure to properly participate in each assigned SKALE
+Chain’s consensus, and maintain uptime and latency standards enforced by the Service Level Agreement (SLA).
+
+Network SLAs are enforced through an algorithmic peer review system whereby each node is appointed 24 node peers to
+monitor and log their network participation, uptime, and latency. These metrics will be collected and averaged on the
+Ethereum mainnet to reward or slash nodes according to their respective performance.
+
+## Requirements
+
+To be added as a node to the SKALE Network, a prospective node must run the SKALE Admin, which manages all operations in the node and is
+installed with the [skale-node-cli](/validators/validator-cli). SKALE Admin evaluates the prospective node to ensure that it's upholding
```

Comment:
`/validators/validator-cli` => believe it is `/ecosystem/validators/node-cli/overview` instead

Reply:
Resolved in [4a45bc9](https://github.com/skalenetwork/docs.skale.space/pull/135/commits/4a45bc9bc4dc5894526497d1df1091ae5836676e)